iroha_error/iroha_error_macro/src/display.rs

```diff
@@ -49,7 +49,11 @@ pub fn impl_fmt(ast: &syn::DeriveInput) -> proc_macro2::TokenStream {
     })
     .map(|(variant, field, fmt)| {
         if field {
-            quote! { Self:: #variant (_) => write!(f, #fmt) }
+            quote! {
+                Self:: #variant (inner) => {
+                    write!(f, "{}. Caused by: {}", #fmt, inner) 
```

Comment:
I guess it shouldn't be like that. It is a purpose of error reporter to print error trace

Reply:
Yeah, sure